core/shared/src/main/scala/sigma/reflection/ReflectionData.scala

```diff
@@ -174,14 +175,6 @@ object ReflectionData {
         mkMethod(clazz, "updateOperations", Array[Class[_]](classOf[Byte])) { (obj, args) =>
           obj.asInstanceOf[AvlTree].updateOperations(args(0).asInstanceOf[Byte])
         },
-        mkMethod(clazz, "getMany", Array[Class[_]](classOf[Coll[_]], classOf[Coll[_]])) { (obj, args) =>
```

Comment:
why removed? 

Where methods from this file are called btw ? ScalaDoc for ReflectionData is obscure

Reply:
they removed because those methods of AvlTree never called via reflection. Interpreter calls them via `xxx_eval` methods. Search for an example `mkMethod(clazz, "getMany_eval",`